website/blog/2021-04-20-tungsten.md

```diff
@@ -0,0 +1,220 @@
+---
+author: Tomasz Godzik
+title: Metals v0.10.2 - Tungsten
+authorURL: https://twitter.com/TomekGodzik
+authorImageURL: https://github.com/tgodzik.png
+---
+
+We're happy to announce the release of Metals v0.10.2, which mainly adds support
+for Scala 3.0.0-RC3 as well a couple of smaller improvements.
+
+<table>
+<tbody>
+  <tr>
+    <td>Commits since last release</td>
+    <td align="center">62</td>
+  </tr>
+  <tr>
+    <td>Merged PRs</td>
+    <td align="center">30</td>
+  </tr>
+    <tr>
+    <td>Contributors</td>
+    <td align="center">8</td>
+  </tr>
+  <tr>
+    <td>Closed issues</td>
+    <td align="center">8</td>
+  </tr>
+  <tr>
+    <td>New features</td>
+    <td align="center">4</td>
+  </tr>
+</tbody>
+</table>
+
+For full details: https://github.com/scalameta/metals/milestone/37?closed=1
+
+Metals is a language server for Scala that works with VS Code, Vim, Emacs,
+Sublime Text and Eclipse. Metals is developed at the
+[Scala Center](https://scala.epfl.ch/) and [VirtusLab](https://virtuslab.com)
+with the help from [Lunatech](https://lunatech.com) along with contributors from
+the community.
+
+## TL;DR
+
+Check out [https://scalameta.org/metals/](https://scalameta.org/metals/), and
+give Metals a try!
+
+- [Scala 3] Support for Scala 3.0.0-RC3
+- Add code action to add/remove string interpolation.
+- Offer organize imports when on an unused import.
+- [Scala 3] Better names for types in completions and hovers.
+
+## Add/remove string interpolation
+
+Thanks to some great work by our newest contributor
+[Giggiux](https://github.com/Giggiux) it's now possible to:
+
+- easily add string interpolation to an existing string
+
+```scala
+val message = "Hello this is a message"
+```
+
+when cursor is positioned inside the string we can invoke the code action and
+get:
+
+```scala
+val message = s"Hello this is a message"
+```
+
+This is a simple action, but together with completions it makes sure that users
+don't waste too much time on it.
```

Comment:
I'm not 100% sure what is meant here, so you may want to rephrase it a bit. After reading it I'm left wondering, "too much time on what?" Maybe something along the lines of:

```suggestion
While this is a simpler code action, it can be paired with other things like completions that allow for a workflow with less manual steps.
```

Reply:
Och, that sounds much better!